unicore/ask/service/models.py

```diff
@@ -1 +1,85 @@
-# models go here
+from uuid import uuid4, UUID
+
+from sqlalchemy import Column, Unicode, Boolean, ForeignKey, Integer
+from sqlalchemy.orm import relationship
+from sqlalchemy_utils import UUIDType
+
+
+from unicore.ask.service import Base
+
+
+class UUIDMixin(object):
+    _uuid = Column(
+        UUIDType(binary=False), name='uuid', default=uuid4,
+        primary_key=True)
+
+    @property
+    def uuid(self):
+        return self._uuid.hex
```

Comment:
Just a note about why I wrote the `UUIDMixin`: `UUID` objects aren't JSON-serializable. But you can query objects using a hex string instead of a `UUID` object. So this is magically making you use the hex string version everywhere in your code.

**end disclaimer**